pandas/io/formats/format.py

```diff
@@ -301,28 +300,26 @@ def _get_footer(self) -> str:
             assert isinstance(
                 self.series.index, (ABCDatetimeIndex, ABCPeriodIndex, ABCTimedeltaIndex)
             )
-            footer += "Freq: {freq}".format(freq=self.series.index.freqstr)
+            footer += f"Freq: {self.series.index.freqstr}"
 
         if self.name is not False and name is not None:
             if footer:
                 footer += ", "
 
             series_name = pprint_thing(name, escape_chars=("\t", "\r", "\n"))
-            footer += (
-                ("Name: {sname}".format(sname=series_name)) if name is not None else ""
-            )
+            footer += f"Name: {series_name}"
```

Comment:
I don't think this is totally equivalent?

```python
>>> series_name = None
>>> f"Name: {series_name}"
'Name: None'
```

Reply:
Take a look to line 305